packages/@aws-cdk/aws-appmesh/lib/tls-validation-context.ts

```diff
@@ -0,0 +1,160 @@
+import * as acmpca from '@aws-cdk/aws-acmpca';
+import { CfnVirtualGateway, CfnVirtualNode } from './appmesh.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
+/**
+ * Represents the properties needed to define TLS validation context
+ */
+export interface TlsValidationContext {
+  /**
+   * Reference to where to retrieve the trust chain.
+   */
+  readonly trust: TlsValidationContextTrust;
+}
+
+/**
+ * All Properties for TLS Validations for both Client Policy and Listener.
+ */
+export interface TlsValidationTrustConfig {
+  /**
+   * VirtualNode CFN configuration for client policy's TLS Validation
+   */
+  readonly virtualNodeClientTlsValidationContextTrust: CfnVirtualNode.TlsValidationContextTrustProperty;
```

Comment:
I would remove "context" from the name of all of these properties:

```suggestion
  readonly virtualNodeClientTlsValidationTrust: CfnVirtualNode.TlsValidationContextTrustProperty;
```

Reply:
addressed.